ansible/playbook.yaml

```
@@ -0,0 +1,68 @@
+- name: Update web servers
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+  - name: trello_api_key
+    prompt: What is your Trello API key?
+    private: false
+
+  - name: trello_api_token
+    prompt: What is your Trello API token?
+    private: false
+
+  tasks:
+  - name: Install latest version of git
+    become : yes
+    ansible.builtin.yum:
+      name: git
+      state: latest
+
+  - name: Install python 3.11
+    become : yes
+    ansible.builtin.yum:
+      name: python311
+      state: latest
```

Not a particular recommendation, but worth noting that we can combine these two steps:
```suggestion
  - name: Install latest version of git & python 3.11
    become : yes
    ansible.builtin.yum:
      name:
        - git
        - python311
      state: latest
```